packages/components/src/HeaderBar/HeaderBar.test.js

```diff
@@ -42,6 +42,16 @@ describe('HeaderBar', () => {
 		expect(element).not.toBeUndefined();
 	});
 
+	it('should render custom Intercom component', () => {
+		function Intercom() {
+			return null;
+		}
+
+		const wrapper = mount(<HeaderBarComponent Intercom={Intercom} />);
```

Comment:
Here you pass the component and not the rendered component. But check my general comment first.

Reply:
Sure, I thought it was more convenient to have it already rendered, but it also makes sense to have the same API than any other injected component 👍